src/Models/Tag.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace CachetHQ\Cachet\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+use Watson\Validating\ValidatingTrait;
+
+class Tag extends Model
+{
+    use ValidatingTrait;
+
+    /**
+     * The fillable properties.
+     *
+     * @var string[]
+     */
+    protected $fillable = ['name'];
+
+    /**
+     * Overrides the models boot method.
+     *
+     * @return void
+     */
+    public static function boot()
+    {
+        parent::boot();
+
+        self::creating(function ($tag) {
+            $tag->slug = Str::slug($tag->name);
```

Comment:
This is probably fine for now. I'll let you know if this changes in Laravel. I'll try to make sure it goes into 5.1 rather than a 5.0 patch release if it is changed.